### DISCUSSION TITLE
LFG!

### DISCUSSION THREAD

**Discussion by Kritoris** [smasherprog/EqTool#112] [Ideas]
A window/interface/something where you can list yourself as LFG and any other toons you are LFG on.

Eg. LFG with a 60 Bard, 56 Shaman, 56 Monk, 49 Rogue, 39 Enchanter.

Bonus points for a freeform/notes section so one could list what they are after. Eg. Plate House for Faction, Chardok for names or even what they don't want. "No KC groups please!"

Technically, the P99 Discord should fill that gap because you can post LFGs there, but it seems to be the place where LFGs go to die. No one ever checks or responds there.

**Comment by smasherprog** [comment]
This will take some time to address so it will be on the backlog for a bit
